MAVProxy/modules/mavproxy_chat/assistant_setup/assistant_file_checker.py

```diff
@@ -64,20 +68,20 @@ def main(openai_api_key=None, delete_unused=False):
         for assistant_file in client.beta.assistants.files.list(assistant_id=assistant.id):
             file_assistant_count[assistant_file.id] = file_assistant_count.get(assistant_file.id, 0) + 1
             filename_size_date = get_filename_size_date_from_id(existing_files, assistant_file.id)
-            print("    id:" + assistant_file.id + " name:" + filename_size_date[0] + " size:" + str(filename_size_date[1]) + " date:" + filename_size_date[2])
+            print("    id:" + assistant_file.id + " name:" + filename_size_date[0] + " size:" + str(filename_size_date[1]) + " date:" + filename_size_date[2])  # noqa

```

Comment:
f-strings can make your code a little more readable:

```suggestion
            print(f"    id:{assistant_file.id} name:{filename_size_date[0]} size:{str(filename_size_date[1])} date:{filename_size_date[2]})  # noqa
```